xtokens/src/lib.rs

```diff
@@ -395,6 +398,39 @@ pub mod module {
 			}
 		}
 	}
+
+	impl<T: Config> XcmTransfer<T::AccountId, T::Balance, T::CurrencyId> for Pallet<T> {
+		fn transfer(
+			who: T::AccountId,
+			currency_id: T::CurrencyId,
+			amount: T::Balance,
+			dest: MultiLocation,
+			dest_weight: Weight,
+		) -> XcmExecutionResult {
+			if amount == Zero::zero() {
+				return Ok(Option::None);
+			}
+			let id: MultiLocation = T::CurrencyIdConvert::convert(currency_id.clone())
```

Comment:
why the clone here?

Reply:
was a leftover after copy pasting this. removed.